### ISSUE TITLE
Intended use with sprockets + cssbundling-rails and jsbundling-rails?

### ISSUE THREAD

**Issue by ivy** [jamesmartin/inline_svg#132]
I see that the `inline_svg` helper is deprecated in favor of `inline_svg_pack_tag` but this seems to require Webpacker. What should we be doing for Rails apps using Hotwire (Stimulus and Turbo) with [cssbundling-rails](https://github.com/rails/cssbundling-rails) and [jsbundling-rails](https://github.com/rails/jsbundling-rails)?

**Comment by jamesmartin** [comment]
👋 @ivy, thanks for opening this issue. We've actually reversed the decision to deprecate the `inline_svg` method (https://github.com/jamesmartin/inline_svg/pull/131). This was going to be deprecated in version 2.0 but we decided that it would be better to allow the gem to be configured to use Sprockets or Webpacker. This should mean fewer code changes for people upgrading from v1.x to v2.x, when that eventually happens.
